js/data.js

```diff
@@ -6,3 +6,14 @@ let data = {
   editing: null,
   nextEntryId: 1,
 };
+
+const previousDataJSON = localStorage.getItem('data-object');
+
+window.addEventListener('beforeunload', function (event) {
+  const dataJSON = JSON.stringify(data);
+  localStorage.setItem('data-object', dataJSON);
+});
+
+if (localStorage.getItem('data-object') !== null) {
```

Comment:
You already have a variable for this, so you should just use `previousDataJSON` instead

Reply:
Thanks Shawn. Updated line 17 on data.js to use variable